### DISCUSSION TITLE
Netbook with cloudready 96.4. Is thoriumOS a possiblility?

### DISCUSSION THREAD

**Discussion by madhits** [Alex313031/ThoriumOS#68] [Q&A]
Hi, I hate to waste your time and mine. but I have a netbook Atom N455 - GMA 3150 graphics that runs amazingly fast on cloudready 96.4 (the FINAL release of CR). Its mostly an offline machine. What I want to know is did thorianOS remove the driver for GMA 3150 acceleration? IE am i supported still? My guess is No, but I had to ask. THANKS

**Comment by Alex313031** [comment]
@madhits No, it should work.
